Zend/zend_vm_def.h

```diff
@@ -3157,6 +3157,30 @@ ZEND_VM_HANDLER(112, ZEND_INIT_METHOD_CALL, CONST|TMPVAR|UNUSED|THIS|CV, CONST|T
 		}
 	}
 
+	if (OP1_TYPE == IS_CV && OP2_TYPE == IS_CONST && fbc->op_array.accessor.type == ZEND_ACCESSOR_GETTER) {
```

Comment:
fbc->op_array.accessor.type may be uninitialized.

$ USE_ZEND_ALLOC=0 valgrind sapi/cli/php ../tests/classes/abstract.phpt
==1580== Conditional jump or move depends on uninitialised value(s)
==1580==    at 0x867E0B6: ZEND_INIT_METHOD_CALL_SPEC_CV_CONST_HANDLER (zend_vm_execute.h:39738)
==1580==    by 0x863826C: execute_ex (zend_vm_execute.h:424)
==1580==    by 0x8638321: zend_execute (zend_vm_execute.h:468)
==1580==    by 0x85EF068: zend_execute_scripts (zend.c:1427)
==1580==    by 0x857F3A9: php_execute_script (main.c:2494)
==1580==    by 0x86A31C0: do_cli (php_cli.c:982)
==1580==    by 0x86A3E2A: main (php_cli.c:1352)